packages/frontend/src/hooks/pivotTable/pivotQueryResults.ts

```diff
@@ -56,10 +56,18 @@ export const pivotQueryResults = ({
     if (rows.length === 0) {
         throw new Error('Cannot pivot results with no rows');
     }
-    const columnOrder = pivotConfig.columnOrder || [];
+
+    const hiddenFieldIds = pivotConfig.hiddenFieldIds || [];
+
+    const columnOrder = (pivotConfig.columnOrder || []).filter((id) => {
+        return !hiddenFieldIds.includes(id);
+    });
+
     // Headers (column index)
     const headerDimensions = pivotConfig.pivotDimensions.filter(
-        (pivotDimension) => metricQuery.dimensions.includes(pivotDimension),
+        (pivotDimension) =>
+            metricQuery.dimensions.includes(pivotDimension) &&
+            !hiddenFieldIds.includes(pivotDimension),
```

Comment:
This still allows pivoted rows to be hidden

Reply:
updated!